homeassistant/components/overkiz/cover_entities/awning.py

```diff
@@ -0,0 +1,69 @@
+"""Support for Overkiz awnings."""
```

Comment:
Is it allowed to have multiple classes in a subfolder? Here we just have 3 classes (base, awning, vertical), however later on for climate devices we will have 10+ implementations.

Reply:
Could it be that `Error: homeassistant/components/overkiz/cover_entities/vertical_cover.py:42: error: Class cannot subclass "OverkizGenericCover" (has type "Any")  [misc]` is related to this?

Reply:
Try adding an `__init__.py` in the subfolder